backend/tracim_backend/lib/cleanup/cleanup.py

```diff
@@ -0,0 +1,339 @@
+import shutil
+import typing
+import uuid
+
+from sqlalchemy.orm import Session
+
+from tracim_backend.applications.share.models import ContentShare
+from tracim_backend.applications.upload_permissions.models import UploadPermission
+from tracim_backend.config import CFG
+from tracim_backend.lib.core.workspace import WorkspaceApi
+from tracim_backend.lib.utils.logger import logger
+from tracim_backend.models.auth import User
+from tracim_backend.models.data import Content
+from tracim_backend.models.data import ContentRevisionRO
+from tracim_backend.models.data import RevisionReadStatus
+from tracim_backend.models.data import UserRoleInWorkspace
+from tracim_backend.models.data import Workspace
+from tracim_backend.models.meta import DeclarativeBase
+
+ANONYMOUS_USER_EMAIL_PATTERN = "anonymous_{hash}@anonymous.anonymous"
```

Comment:
Maybe use domain that we are absolutely sure that never exist (to never send mail by error).

```suggestion
ANONYMOUS_USER_EMAIL_PATTERN = "anonymous_{hash}@anonymous.local"
```